Objects/src/EventBase.cc

```diff
@@ -214,6 +219,9 @@ panda::EventBase::doGetEntry_(TTree& _tree)
   if (triggerObjects.size() != 0 && run.hlt.filters)
     triggerObjects.makeMap(*run.hlt.filters);
 
+  if (rng) 
+    rng->generate(eventNumber);
```

Comment:
We should make it somehow possible to turn off the resetting at every event. As you pointed out before, that might not even be the default behavior we want. If someone new uses the feature not for syncing, we should give them the most random numbers possible. In general, I also asked if there was interest in this feature in order to avoid completely unrepeatable numbers like timestamps, as well as even less correct constant numbers. I'll put a strict outline of default behaviors that I think would be nice in the main conversation.